matterbridge.go

```diff
@@ -89,10 +90,19 @@ func (b *Bridge) handleMatter() {
 		case "!gif":
 			message.Text = b.giphyRandom(strings.Fields(strings.Replace(message.Text, "!gif ", "", 1)))
 			b.Send(b.Config.IRC.Nick, message.Text)
+		case "!priv":
+			who := strings.Fields(message.Text)[1]
```

Comment:
Need to check if len(result) of strings.Fields == 2 otherwise you can get nil pointer panics